.gitmodules

```diff
@@ -1,6 +1,8 @@
 [submodule "qemu"]
 	path = qemu
-	url = https://github.com/pranith/qemu
+	url = https://github.com/B-Bimmermann/qemu-1.git
+	branch = qsim_v25_qemu_26
```

Comment:
Can you remove this change? Once I merge your branch into my repository, this will not be necessary.